VERSION

```diff
@@ -1 +1 @@
-v4.2.7
+4.2.8
```

Comment:
We dont need the v to be here, right?

Reply:
Anyway, I'd to have this in now, for my next PR, if this is not good, we can fix it for 4.2.9